### PR TITLE
Added Wayback Machine Search Site inside extension with suggestion feature

### PR DESCRIPTION
**Pull request by collinx**
This PR integrates the Wayback Machine's Site Search into the extension (showing suggestions).
Below you can see the working of this feature.

sitesearchb

@rchrd2  @markjohngraham  Kindly review this PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +56,6 @@
     <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
   </body>
 </html>
-<script src="scripts/jquery.js"></script>
 <script src="scripts/popup.js"></script>
 <script src="scripts/bootstrap.min.js"></script>
```
Comment (rgpt) [inline index.html]: in console if you see bootstrap.min.js is showing error as "bootstrap's javascript needs jquery " and we do not want that as jquery can make extension slow so you may have to remove bootstrap's js

Comment (collinx) [inline index.html]: @rgpt thanks for taking the time to go through my code and reviewing it. As for your review, I have already seen the error and it is a custom exception which is thrown if jquery is not there. Even if it is being shown on the console, it doesn't have any effect on speed or efficiency of extension.

**Comment by abhidas17695** [comment]
If the user were to type in a word and click on the search button directly , instead of creating a new tab displaying the the list of all websites archives related to the word , as seen on the site search , the new tab is displaying the search results of the URL http://key_word , where key_word is the word typed in. 

Instead of this : 
capturecollinx1


this is being displayed : 

capturecollinx2

This is not optimum functionality according to me. I hope I am not being unnecessarily harsh. I am talking from the point of view of a user.

**Comment by collinx** [comment]
@abhidas17695 thank you for your point of view. I can totally agree with you on this point. If you have gone through the code, I had used `web.archive.org` domain before which I have rectified to `web-beta.archive.org` on reviewing. Your point really helped in seeing this fault.
Now, if you will search for the same, it will refer to correct URL.
Lastly, giving a review as a user or as a fellow developer can't be harsh. It just reflects the person's will to contribute to the codebase. Reviews are always welcome here ;)

**Comment by abhidas17695** [comment]
It is working perfectly now. Good job

**Comment by collinx** [comment]
@rchrd2 @markjgraham Kindly review this PR. Added the new changes to the branch v1.3.3 here. Recursive Searching is working fine alongside the social buttons.

**Comment by collinx** [comment]
@rakesh-chinta I have removed conflicts. See to this PR too.

**Pull request merged**